res/controllers/Hercules_DJControl_Inpulse_500.midi.xml

```diff
@@ -0,0 +1,3424 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
_(Placing this comment at the top, though this applies to the whole file)._

The controller mapping XML files should be formatted with an indentation of 4 spaces (though some existing files deviate from this and are indented with 2 spaces instead).

This file currently contains a mix of tabs and spaces as well as varying indentation.